Session-3/CurrencyExchangeApp/CurrencyConverter/ConvertCurrency.cs

```diff
@@ -0,0 +1,44 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+using RestSharp;
+using Newtonsoft.Json;
+
+
+namespace CurrencyConverter
+{
+    public class ConvertCurrency
```

Comment:
Give proper names for namespace and class.
Something like CurrencyExchange and CurrencyConverter